QUEST.tsv

```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
-QUEST_20150317_001166	However, Jigufuri who was the other party that usually felt that it is possible to win with certainty on this occasion, that the next you feel that it is not by even less money to get the word one head, and as long as there is money more than anything else, Because had never refused to bet, I would accept the finally the bet. {nl} Of course, this bet was over in the course model Stars' victory. {nl}
-QUEST_20150317_001167	Jigufuri Mr. and betting begins, did his Byorujito to beat soothe the words that do not move an inch, it is not visible even signs of moving at all. If {nl} eventually Jigufuri Mr. there is no way words are repeated the words is greater than or equal to stretch, to pass the reins of the horse listed the idea slow to promise to throw in the model Stars san, dry spare time also has excellent beginning to die without was. {nl} and Jigufuri Mr. disappears, model Stars' mood was not good.
-QUEST_20150317_001168	Since the usual model Stars Mr. was the one who does not hurt the mood or lose even become cheerful and if you win the bet, strange feelings of feeling even win the bet model Stars Mr. is bad for the first time experience really born Could. {nl}
-QUEST_20150317_001169	Model Stars' as softening close it fooled Jigufuri Mr. friends be the same, even did skip the opportunity to earn a lot of money, and walked slowly gathered the words named after that it felt the same mixed feelings. {nl} model Stars Mr. while walking, this time there was the fact that Kai defeated unconditionally bet. So in "no matter what, bet this time,'ll not over silver one or more." It was done so decided.
-QUEST_20150317_001170	People who model Stars Mr. encounters that were walking down the street and pull to say so {nl} was the soldiers of the people number. Soldiers to {nl} model Stars san see looked to the hassle to put something. {nl} model Stars Mr., was Jinagaryo only whether they are not, soldiers cried when you see the model's Stars. {nl}
-QUEST_20150317_001171	There will pull the words to Uncle. Come, do not me help us. {nl} model Stars Mr., the number of the soldiers did not know what helped me in any thing, the sound to hear it went closer to them to collect the words, rather than be able to pass as it is. {nl} Once the model Stars Mr. approaches, one of the soldiers said. {nl} "just good come small. Cattle Once you have me look at the referee that we were one of the bet here." {nl}
+QUEST_20150317_001158	Anyhow, as Modestas and the Old Lady parted ways, he knew he now had a dangerous power guaranteeing his victory at any time in the near future. {nl} He walked, looking for the next person to bet huge on. As he wondered how much to bet on his next move, he heard the sound of running footsteps behind him.
```

Comment:
It says "footsteps" here, but the next line says Sigfried is riding on a horse. Shouldn't it be something along the lines of "sound of hooves" instead?